src/rpc/server/scala/src/main/scala/com/ctc/omega_edit/grpc/Editors.scala

```diff
@@ -57,7 +57,7 @@ object Editors {
   }
 
   private def idFor(path: Option[Path]): String = path match {
-    case None    => UUID.randomUUID().toString.take(8)
+    case None    => UUID.randomUUID().toString
```

Comment:
No need to truncate the UUID to 8 characters.  The C++ server uses 36 character UUIDs for Session and Viewport IDs.